src/rofl/common/crofbase.cc

```diff
@@ -743,8 +744,8 @@ void
 crofbase::handle_connect_refused(
 		crofdpt& dpt, crofconn& conn)
 {
-	journal.log(LOG_INFO, "connection refused: dptid=%s, auxid=%s, raddr=%s",
-			dpt.get_dptid().str().c_str(), conn.get_auxid().str().c_str(), conn.get_raddr().str().c_str());
+	journal.log(LOG_INFO, "connection refused: dptid=0x%" PRIx64 ", auxid=%s, raddr=%s",
```

Comment:
as we discussed, please check the output of `dptid=0x%" PRIx64 "` , also this is repeated below